Vagrantfile

```diff
@@ -68,6 +70,7 @@ Vagrant.configure(2) do |config|
 	  ansible.playbook = "site.yml"
 	  ansible.extra_vars = "globals.yml"
 	  ansible.inventory_path = INVENTORY_FILE
+	  ansible.vault_password_file = "~/.vault_pass.txt"
```

Comment:
"vagrant up" failed because it doesn't find this file, I guess this is specific to the local setup. 
may be better to use "--ask-pass" here instead since it will be more generic ?